src/Microsoft.DotNet.Tools.Publish/README.md

```diff
@@ -3,53 +3,74 @@ dotnet-publish
 
 **NAME**
 
-dotnet-publish -- packs the application and all of its dependencies into a folder getting it ready for publishing
+`dotnet-publish` 
+> packs the application and all of its dependencies into a folder getting it ready for publishing
 
 **SYNOPSIS**
 
-dotnet-publish [options] [project]
+`dotnet-publish [options] [project]`
 
 **DESCRIPTION**
-dotnet-publish will compile the application, read through its dependencies specified in project.json and publish the resulting set of files to a directory. This directory contains the assemblies, the runtime as well as the runnable version of the application. This directory can then be moved to a different machine and the application will be able to be ran regardless of existence of any other runtime.  
 
-dotnet-publish needs access to project.json to work. If it is not specified on invocation via [project], project.json in the current directory will be the default. If no project.json can be found, dotnet-publish will error out. 
+`dotnet-publish` will compile the application, read through its dependencies specified in _project.json_ and publish the resulting set of files to a directory. 
+This directory contains the assemblies, the runtime as well as the runnable version of the application. 
+This directory can then be moved to a different machine and the application will be able to be ran regardless of existence of any other runtime.  
 
-The command also requires information on the targeted framework and runtime, both of which can be specified on the command line. If the runtime is not specified, the command will default to the runtime for the current operating system. If the framework is not specified, the command will read the information from the project.json file. In case of no valid framework found, the command will error out. In case of multiple valid frameworks found, the command will publish for all valid frameworks. 
+`dotnet-publish` needs access to _project.json_ to work. 
+If it is not specified on invocation via [project], _project.json_ in the current directory will be the default. 
+If no _project.json_ can be found, `dotnet-publish` will error out. 
+
+The command also requires information on the targeted framework and runtime, both of which can be specified on the command line. 
+If the runtime is not specified, the command will default to the runtime for the current operating system. 
+If the framework is not specified, the command will read the information from the _project.json_ file. 
+In case of no valid framework found, the command will error out. 
+In case of multiple valid frameworks found, the command will publish for all valid frameworks. 
 
 
 * ubuntu
```

Comment:
done

Lee Campbell

E: Lee.Ryan.Campbell@gmail.com

On 3 December 2015 at 18:58, Zlatko Knezevic <notifications@github.com>
wrote:

> In src/Microsoft.DotNet.Tools.Publish/README.md
> <https://github.com/dotnet/cli/pull/380#discussion_r46594349>:
>
> >
> > -The command also requires information on the targeted framework and runtime, both of which can be specified on the command line. If the runtime is not specified, the command will default to the runtime for the current operating system. If the framework is not specified, the command will read the information from the project.json file. In case of no valid framework found, the command will error out. In case of multiple valid frameworks found, the command will publish for all valid frameworks.
> > +`dotnet-publish` needs access to _project.json_ to work.
> > +If it is not specified on invocation via [project], _project.json_ in the current directory will be the default.
> > +If no _project.json_ can be found, `dotnet-publish` will error out.
> > +
> > +The command also requires information on the targeted framework and runtime, both of which can be specified on the command line.
> > +If the runtime is not specified, the command will default to the runtime for the current operating system.
> > +If the framework is not specified, the command will read the information from the _project.json_ file.
> > +In case of no valid framework found, the command will error out.
> > +In case of multiple valid frameworks found, the command will publish for all valid frameworks.
> >
> >
> >  * ubuntu
>
> Just goes to show that he who works makes mistakes. :) Remove it, push
> again and we should merge. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/cli/pull/380/files#r46594349>.
>